frontend/app/api/zoom/generateToken.ts

```diff
@@ -0,0 +1,61 @@
+import axios from 'axios';
+import { NextResponse } from "next/server";
+import { NextApiRequest, NextApiResponse } from 'next/types';
+import { PrismaClient} from "@prisma/client";
+import qs from 'query-string';
+
+const prisma = new PrismaClient();
+
+// function to generate zoom access token 
+const generateZoomToken = async () => {
+  try {
+    const clientId = process.env.NEXT_PUBLIC_ZOOM1_CLIENT_ID;
+    const clientSecret = process.env.NEXT_PUBLIC_ZOOM1_CLIENT_SECRET;
+    const accountId = process.env.NEXT_PUBLIC_ZOOM1_ACCOUNT_ID;
+  
+    // check if the environment variables are set
+    if (!clientId || !clientSecret || !accountId) {
+      throw new Error('Environment variables not set');
+    }
+
+    // make a post req to zoom api 
+    const request = await axios.post(
+      'https://zoom.us/oauth/token', 
+      qs.stringify({ grant_type: 'account_credentials', account_id: accountId }), 
+      {
+        headers: {
+          'Content-Type': 'application/x-www-form-urlencoded',
+          'Authorization': `Basic ${Buffer.from(`${clientId}:${clientSecret}`).toString('base64')}`,
+        }
```

Comment:
i think eventually we will create a secret in .env for the link "https://zoom.us/oauth/token" but this is good for now